core/client/src/main/java/alluxio/client/FileSystemTestUtils.java

```diff
@@ -37,10 +37,11 @@
    * @param fileName the name of the file to be created
    * @param len file size in bytes
    * @param options options to create the file with
-   * @throws IOException if {@code path} is invalid (e.g., illegal URI)
+   * @throws IOException if an IO exception occurs
+   * @throws AlluxioException if an Alluxio exception occurs
    */
   public static void createByteFile(FileSystem fs, String fileName, int len,
-      CreateFileOptions options) throws IOException {
+      CreateFileOptions options) throws IOException, AlluxioException {
```

Comment:
should this class be in `test` instead of `main` if it is test utils? the reason I ask is because I would prefer to simplify the signature to `throws Exception` which is common-place in our test code base